packages/react-slider/src/components/Slider/renderSlider.tsx

```diff
@@ -11,18 +11,9 @@ export const renderSlider = (state: SliderState) => {
 
   return (
     <slots.root {...slotProps.root}>
-      {state.marks && <slots.marksWrapper {...slotProps.marksWrapper} />}
-      <slots.sliderWrapper {...slotProps.sliderWrapper}>
-        <slots.rail {...slotProps.rail} />
-        <slots.trackWrapper {...slotProps.trackWrapper}>
-          <slots.track {...slotProps.track} />
-        </slots.trackWrapper>
-        <slots.thumbWrapper {...slotProps.thumbWrapper}>
-          <slots.thumb {...slotProps.thumb} />
-        </slots.thumbWrapper>
-        <slots.activeRail {...slotProps.activeRail} />
-        <slots.input {...slotProps.input} />
-      </slots.sliderWrapper>
+      <slots.rail {...slotProps.rail} />
+      <slots.thumb {...slotProps.thumb} />
+      <slots.input {...slotProps.input} />
```

Comment:
The input is hidden right?  Order probably doesn't matter here, but would it be better if it were behind the rail and thumb?

Reply:
I moved it to the back and insured that click events go all the way to it. Probably safer to do it that way.